rest_registration/api/views/register_email.py

```diff
@@ -80,8 +81,11 @@ def verify_email(request):
     '''
     Verify email via signature.
     '''
-    process_verify_email_data(
+    user = process_verify_email_data(
         request.data, serializer_context={'request': request})
+
+    signals.user_changed_email.send(sender=None, user=user, request=request)
```

Comment:
We should also add signal `.send` in `register_email` in the if branch where `registration_settings.REGISTER_EMAIL_VERIFICATION_ENABLED` is `False`.